mkdocs.yml

```diff
@@ -144,6 +144,17 @@ pages:
       - FAQ: synthetics/faq.md
     - Integrations:
       - Overview: integration/index.md
+      - Apache: integration/apache-integration.md
```

Comment:
@fulyauluturk if all these were added here, shouldn't the mkdocs.yml diff show these removed from Logs and Monitoring sections?

Reply:
We said we won't be removing the old ones but trim the pages and link to the new page. That's why I didn't remove them from mkdocs.

Reply:
Ah, right.